content/sql/concepts/operators/terms/null-safe-equal/null-safe-equal.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'NULL SAFE EQUAL'
+Description: 'Performs an inequality comparison similar to the '=' operator, using the logic of an XNOR gate. Can accept NULL as a value and will not output NULL.'
+Subjects: 
+  - 'Computer Science'
+Tags: 
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'Tables'
+CatalogContent: 
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **'NULL SAFE EQUAL'** operator performs an equality comparison like the '=' operator. However it can also accept NULL values and never returns NULL, instead always returning TRUE or FALSE. It uses the logic of an XNOR gate to determine the value returned.
+
+The 'NULL SAFE EQUAL' function is useful for when an XNOR gate that can accept NULL values and always returns True or False is needed.
```

Comment:
```suggestion
The `NULL SAFE EQUAL` operator is useful when an XNOR-like logic is required for comparisons that can accept NULL values and always return True or False.
```
The NULL SAFE EQUAL is an operator so it must be enclosed in `.....` backticks, also removed the function word as it is not a function.
Also, NULL SAFE EQUAL operator is useful when XNOR-like logic is required not literally an XNOR gate.